master.js

```diff
@@ -0,0 +1,356 @@
+/* eslint-disable max-len */
+const { Traffic, GAwesomeClient } = require("./Modules");
+const { Boot, Sharder } = require("./Internals");
+const { Stopwatch } = require("./Modules/Utils");
+const centralClient = new GAwesomeClient(null);
+
+const database = require("./Database/Driver.js");
+const auth = require("./Configurations/auth.js");
+const configJS = require("./Configurations/config.js");
+const configJSON	= require("./Configurations/config.json");
+const configWarnings = [];
+const figlet = require("figlet");
+
+const scope = { safeMode: false };
+
+Boot({ configJS, configJSON, auth }, scope).then(() => {
+	if (scope.migrating) return;
+	logger.info("Connecting to MongoDB...", { URL: configJS.database.URL, db: configJS.database.db });
+
+	// eslint-disable-next-line promise/catch-or-return
+	database.initialize(configJS.database).catch(err => {
+		logger.error(`An error occurred while connecting to MongoDB! x( Is the database online?`, { config: configJS.database }, err);
+		process.exit(1);
+	}).then(async () => {
+		const db = database.getConnection();
+
+		if (db && !scope.safeMode) {
+			await logger.info(`Connected to MongoDB successfully.`);
+
+			logger.silly("Confirming MongoDB config values.");
+			const adminDatabaseResponse = await db.client.admin().command({ getCmdLineOpts: 1 });
```

Comment:
This WILL THROW if we cannot access the admin DB. And truth be told, at that point it's safe to assume you are not dumb enough to expose your db